src/codeu/chat/common/VersionInfo.java

```diff
@@ -0,0 +1,46 @@
+package codeu.chat.common;
+
+import java.io.IOException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import codeu.chat.util.Uuid;
+
+/**
+ * Holds the current version of the CodeU chat server.
+ * 
+ * For clients, this can be used to determine whether or not the server version
+ * matches that of the client. For servers, this is used to advertise the
+ * version of the server.
+ */
+public class VersionInfo {
+  private static final String CURRENT_VERSION = "1.0.0";
+
+  private Uuid version;
```

Comment:
It can't be final with the current code because the compiler counts the assignment in the catch statement as a reassignment.